openVpnConfig/ap-northeast-1.conf

```diff
@@ -0,0 +1,181 @@
+client
+dev tun
+proto udp
+remote ap-northeast-1.vpn.1nce.com 1194
+resolv-retry infinite
+nobind
+explicit-exit-notify 3
+keepalive 5 30
+user root
+group nogroup
+remote-cert-tls server
+persist-key
+persist-tun
+verb 3
+auth-nocache
+auth-user-pass /etc/openvpn/credentials.txt
```

Comment:
To avoid testing the full roll-out I would suggest adding double quotes around the credentials path to have the same syntax then all the other openVpnConfig files.